src/ActionModal/components/DelegationModal.vue

```diff
@@ -310,7 +314,10 @@ export default {
   validations() {
     return {
       amount: {
-        required: x => !!x && x !== `0`,
+        required: x =>
+          this.network === "polkadot" && this.totalStaked > 0
```

Comment:
shouldn't it be `this.network.startsWith("polkadot")``?

Reply:
Agree!